android/res/values-zh/strings.xml

```diff
@@ -19,16 +19,16 @@
  */
 -->
 <resources>
-    <string name="you_dont_need_a_vpn_to_use_frostwire_bullet_html"><![CDATA[披露： 虽然建议您下载， <b>您不需要VPN使用FrostWire。 </b>]]></string>
-    <string name="tip_touch_tags_to"><![CDATA[ <b>信息:</b> 触摸应用的标签]]></string>
-    <string name="unprotected_connections_visibility_bullet_html"><![CDATA[ <b>您的隐私是一项基本人权</b>. 不受保护的连接意味着您在线进行的所有操作（包括<b>下载，搜索，消息传递</b>）都可以被网络上的任何人监控和拦截，例如： 互联网服务提供商，黑客，政府。<br><br><b>我们强烈建议您使用VPN服务</b>. <br> 以下是一些提供可靠服务并与FrostWire配合良好的选项：]]></string>
+    <string name="you_dont_need_a_vpn_to_use_frostwire_bullet_html"><![CDATA[公开的秘密： 虽然建议您下载， <b>您不需要VPN使用FrostWire。 </b>]]></string>
+    <string name="tip_touch_tags_to"><![CDATA[ <b>提示:</b> 触摸以应用标签]]></string>
```

Comment:
![Uploading Screen Shot 2022-01-29 at 8.41.37 AM.png…]()

Seems to me it was better before.

Reply:
sorry about this，good check